src/Mono.Android/Android.Runtime/Logger.cs

```diff
@@ -58,12 +58,19 @@ public static void Log (LogLevel level, string appname, string? log) {
 			}
 		}
 
+		#if !NATIVEAOT
 		[DllImport (RuntimeConstants.InternalDllName, CallingConvention = CallingConvention.Cdecl)]
 		extern static uint monodroid_get_log_categories ();
+		#endif
 
 		static Logger ()
 		{
+			#if NATIVEAOT
+			// TODO: p/invoke into __system_property_get
+			Categories = LogCategories.Default | LogCategories.Assembly;
```

Comment:
wrt [my earlier comment](https://github.com/dotnet/android/pull/9624/files#r1887733483):

> I'd really prefer the need for any such changes to `Mono.Android.dll`, if possible…

Could we instead have `LogCategories` (or an `int` variation) be a member of `JnienvInitializeArgs`, and have *both* NativeAOT and MonoVM use `JniEnvInit.Initialize(JnienvInitializeArgs*)`?  This would allow removing the need for `monodroid_get_log_categories()` entirely, and aim for consistent initialization between the two environments.

Reply:
…especially when `logCategories` *is already a field* in `JnnenvInitializeArgs` (?!):

https://github.com/dotnet/android/blob/6b91b0423dab56907b1d1dc41677532f7e44f8e3/src/Mono.Android/Android.Runtime/JNIEnvInit.cs#L24

Why do we have this P/Invoke at all?!

Reply:
I’ll open a new PR to remove the p/invoke. I’ll see if we still need a define later.

Reply:
It seems like this will work:
* https://github.com/dotnet/android/pull/9625

But we'll want to merge https://github.com/dotnet/android/pull/9622 first, or the `pinvoke-table` will have conflicts.